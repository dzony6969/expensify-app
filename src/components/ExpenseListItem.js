import React from 'react'
import { Link } from 'react-router-dom';
import moment from 'moment'
import numeral from 'numeral'




const ExpenseListItem = ({description, amount, createdAt, id }) => (
    <div>
     <Link to={`/edit/${id}`} >
      <h2>{description}</h2>
      </Link>
      <p> 
    {numeral(amount /100).format('0,0.00') + 'z≈Ç '}
       - 
    {moment(createdAt).format('Do MMMM, YYYY')}
      
      </p>
      
    </div>
);

export default (ExpenseListItem);
