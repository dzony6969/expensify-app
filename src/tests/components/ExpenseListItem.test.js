import ExpenseListItem from '../../components/ExpenseListItem'
import { shallow } from 'enzyme';
import React from 'react'
import expenses from '../fixtures/expenses';

test('should render each list item', () =>{
    const wrapper = shallow(<ExpenseListItem {...expenses[0]}/>)
    expect(wrapper).toMatchSnapshot();

});