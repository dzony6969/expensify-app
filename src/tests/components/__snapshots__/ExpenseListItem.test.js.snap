// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render each list item 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/1"
  >
    <h2>
      gum
    </h2>
  </Link>
  <p>
     
    123
     - 
    0
     
  </p>
</div>
`;
